Makefile

```diff
@@ -49,6 +53,16 @@ core: Holmakefiles
 main: Holmakefiles
 	cd $(SRCDIR) && $(HOLMAKE)
 
+tests: main $(TESTS)
+	@echo "===================================================="
+	@echo "====     all HolBA tests ran successfully       ===="
+	@echo "===================================================="
+	@echo ""
+
+$(TESTS):
```

Comment:
The rule dependencies are incorrect: it fails if you run `make -j8 tests`. That doesn't matter for the CI, because it only uses one job, but we should be aware of it. One quick workaround that should be enough (let's see) is the following:

```makefile
tests: $(TESTS)

$(TESTS): main
```